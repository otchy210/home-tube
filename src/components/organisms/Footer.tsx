import React from 'react';

const Footer: React.FC = () => {
    return (
        <footer className="bg-dark text-muted small text-white-50 mt-3 py-1 px-3">
            HomeTube is written by{' '}
            <a href="https://twitter.com/otchy" target="_blank">
                @otchy
            </a>
            . Find more info in the{' '}
            <a href="https://github.com/otchy210/home-tube" target="_blank">
                GitHub page
            </a>
            .
        </footer>
    );
};

export default Footer;
