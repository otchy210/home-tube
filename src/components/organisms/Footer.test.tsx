import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
    it('renders properly', () => {
        const { container } = render(<Footer />);
        expect(container).toMatchSnapshot();
    });
});
