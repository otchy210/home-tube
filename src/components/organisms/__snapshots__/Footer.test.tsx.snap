// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
<div>
  <footer
    class="bg-dark text-muted small text-center text-white-50 mt-3 py-1 px-3"
  >
    HomeTube is written by
     
    <a
      href="https://twitter.com/otchy"
      target="_blank"
    >
      @otchy
    </a>
    . Find more info in the
     
    <a
      href="https://github.com/otchy210/home-tube"
      target="_blank"
    >
      GitHub page
    </a>
    .
  </footer>
</div>
`;
